.chloggen/telemetrygen-traces-fix-service-name-and-large-payloads.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: telemetrygen
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fixes issue where service name option didn't properly set on the trace, and fixes issue where large trace payloads (>2000) wouldn't generate the correct number of traces.
```

Let's remove the large payloads mention and revert these changes

```suggestion
note: `telemetrygen traces` now correctly sets the `service.name` resource attribute on generated traces
```